nautobot_ssot/models.py

```diff
@@ -161,7 +161,7 @@ class SyncLogEntry(BaseModel):
     synced_object_id = models.UUIDField(blank=True, null=True)
     synced_object = GenericForeignKey(ct_field="synced_object_type", fk_field="synced_object_id")
 
-    object_repr = models.CharField(max_length=200, blank=True, default="", editable=False)
+    object_repr = models.CharField(max_length=511, blank=True, default="", editable=False)
```

Comment:
why 511, in concrete?

Reply:
Taken from another field on the model, I assumed that there was a reasoning behind it because of 512-1=511:

```
message = models.CharField(max_length=511, blank=True)
```

Reply:
maybe @glennmatthews could enlighten us about it?

Reply:
Is this something you'd like to see as part of this PR or rather a separate issue to open going forward?

Reply:
If we're changing the model anyway, let's change it to the right thing.